src/libraries/Native/Unix/System.Security.Cryptography.Native.Android/pal_ecc_import_export.c

```diff
@@ -331,7 +331,7 @@ static jobject AndroidCryptoNative_CreateKeyPairFromCurveParameters(
     {
         // Destroy the private key data.
         (*env)->CallVoidMethod(env, loc[privateKey], g_destroy);
-        CheckJNIExceptions(env); // The destroy call might throw an exception. Clear the exception state.
+        TryClearJNIExceptions(env); // The destroy call might throw an exception. Clear the exception state.
```

Comment:
When intentionally ignoring return values I find it helpful to cast the return to `(void)` so it is clear the return is intentionally ignored. For example,

```c
(void)TryClearJNIExceptions(env);
```